src/overlays/actors/ovl_Bg_Breakwall/z_bg_breakwall.c

```diff
@@ -46,9 +46,15 @@ void func_808B7D34(Actor* thisx, PlayState* play);
 void BgBreakwall_Draw(Actor* thisx, PlayState* play);
 
 ActorInit Bg_Breakwall_InitVars = {
-    ACTOR_BG_BREAKWALL, ACTORCAT_ITEMACTION,           FLAGS,
-    GAMEPLAY_KEEP,      sizeof(BgBreakwall),           (ActorFunc)BgBreakwall_Init,
-    (ActorFunc)NULL,    (ActorFunc)BgBreakwall_Update, (ActorFunc)NULL,
+    ACTOR_BG_BREAKWALL,
```

Comment:
Doesn't look like this needs the comments, but probably better to add them for consistency.